frontend/static/js/process-editor.js

```diff
@@ -267,8 +267,8 @@ function checkCorrectInputs() {
     names.forEach(element => {
         const inputs = document.getElementsByName(element);
```

Comment:
Kann die if nicht vor die Schleife gezogen werden - muss ja nicht für jedes mal wieder geprüft werden, oder?

```suggestion

if (element === 'target-average' || element === 'target-minimum' || element === 'target-maximum') {
        const inputs = document.getElementsByName(element);
        for (let i = 0; i < inputs.length; i++) {
             helper.addMinMaxPopup(inputs[i]);
        }
```